doc/development-workflow.md

```diff
@@ -0,0 +1,22 @@
+# Development workflow
+
+## PR author
+
+1. Open a PR with the changes you want to merge.
+
+2. Add the "Waiting on factcheck" label if the changes require feedback from someone other than developers.
+
+3. Move the associated Trello card from "In progress" to "Code review".
```

Comment:
@chrissross: thank you for sharing this document. It is true that we have not sorted out yet how to organize labels and workflow. 

Another option could be following the convention of having the tags described below before deploying to production:

##### If it is not a refactoring (need fact-check)
1. Label with LGTM
2. Label with Fact-checked

##### If it is a refactoring (no fact-check needed)
1. Label with LGTM
2. Label with Refactoring

A draft of the process could be something like:

1. Open a PR with the changes you want to merge.
2. Add the “Refactoring” label if the changes does not require feedback from someone other than developers.
3. Move the associated Trello card from "In progress" to "Code review".
4. Once reviewed, If it is a refactoring, move to ready for deploy.
5. Once reviewed, If it is not a refactoring, move the card to the Fact-check
6. Once fact-checked, move the card to ready for deploy.